sauce/features/accounts/customFlagNames/settings.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  name: 'CustomFlagNames',
+  type: 'checkbox',
+  default: false,
+  section: 'accounts',
+  title: 'Adds Custom Flag Names',
+  description: 'Adds the ability to change set custom flag names. **WARNING** Flag names are stored in localStorage and will be lost if localStorage is cleared.'
```

Comment:
You need to use "back tics" for the text to be recognized as markdown. Like this \`Hello **world**\`